utils/docker/run-build.sh

```diff
@@ -111,7 +111,7 @@ cmake .. -DDEVELOPER_MODE=1 \
 
 make -j2
 ctest --output-on-failure --timeout 540
-if [ "$COVERAGE" = "1" ]; then
+if [ "$COVERAGE" == "1" ]; then
```

Comment:
"man test" (test is an alias for "[" command) shows that == is not supported
have you verified that it works?

Reply:
it is bash extension and for strings it's the same as "=", see "man bash"

Reply:
== works with [[, which is a bash extension, but I'm asking about [

Reply:
I think it's ok for both: https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html#Bash-Conditional-Expressions

Reply:
This document is not clear about that. 

But I dug deeper into bash documentation. This syntax works only because bash overrides [ command with a builtin that accepts both styles.
So is this change correct? Yes, it is.
Is it a "correction"? No, the code was fine before.